Source/Core/Core/HW/SystemTimers.cpp

```diff
@@ -132,11 +125,25 @@ void IPC_HLE_UpdateCallback(Core::System& system, u64 userdata, s64 cyclesLate)
   }
 }
 
+void GPUSleepCallback(Core::System& system, u64 userdata, s64 cyclesLate)
+{
+  auto& core_timing = system.GetCoreTiming();
+  system.GetFifo().GpuMaySleep();
+  core_timing.ScheduleEvent(VideoInterface::GetTicksPerField() / 8 - cyclesLate, et_GPU_sleeper);
+}
+
+void PerfTrackerCallback(Core::System& system, u64 userdata, s64 cyclesLate)
+{
+  auto& core_timing = system.GetCoreTiming();
+  g_perf_metrics.CountPerformanceMarker(system, cyclesLate);
+  core_timing.ScheduleEvent(GetTicksPerSecond() / 5 - cyclesLate, et_perf_tracker);
```

Comment:
Why was 5 chosen?

Reply:
I tested a bunch of values, and having this update at 5 times per second gives a good balance of reducing noise and looking smooth. I really should have made a circular buffer so it could update every frame, but I was lazy.